### ISSUE TITLE
Stuck in "rendering geoJSON"

### ISSUE THREAD

**Issue by mmd-osm** [tyrasd/overpass-turbo#367]
Some queries produce really bad data, triggering error messages like "Error: Invalid LatLng object: (NaN, NaN)" on the javascript console while rendering geoJSON. Unfortunately, overpass turbo gets stuck here forever. It would be better to see at least some of the data (and optionally reporting that there was some issue rendering the result).

Example:

```
[adiff:"2018-02-13T20:53:00.000Z","2018-02-13T20:54:00.000Z"];
way(561040198);
(._; >;);
out meta geom;
```

**Comment by tyrasd** [comment]
I agree, that overpass turbo shouldn't crash on invalid input. But isn't this primarily (?) a bug of the overpass api: https://github.com/drolbr/Overpass-API/issues/463 (falsely returning a deleted way version as not-deleted)

**Comment by mmd-osm** [comment]
>  isn't this primarily (?) a bug of the overpass api

Yes, exactly, that's what I meant with "Some queries produce really bad data"... :)

**Comment by gy-mate** [comment]
I've just ran into this with the following query:

```
way(680443640) ->.searchArea;
nwr(area.searchArea);
out;
```

But it's still the same with the query mentioned in the issue description.

**Comment by mmd-osm** [comment]
@gy-mate : I cannot reproduce the issue. On which Overpass instance are you running your query?

**Comment by gy-mate** [comment]
@mmd-osm I'm using `2024-01-15/8db5c43` on www.overpass-turbo.eu with Safari `17.3.1`. It worked for me once a few minutes ago, but it's not working again:

<img width="40%" height="40%" alt="Screenshot 2024-02-13 at 18 26 48" src="https://github.com/tyrasd/overpass-turbo/assets/13014176/36146a85-e0a3-4b2f-9f57-40dbe9b60d67">

The following error appears in the console just when it starts _rendering geoJSON_:
```
[Error] TypeError: undefined is not an object (evaluating 'e.lat')
	project (urlParameters-2b23d0ae.js:16:14665)
	latLngToPoint (urlParameters-2b23d0ae.js:16:13266)
	(anonymous function) (urlParameters-2b23d0ae.js:290:38772)
	eachLayer (urlParameters-2b23d0ae.js:16:72072)
	_onZoomEnd (urlParameters-2b23d0ae.js:290:38126)
	onAdd (urlParameters-2b23d0ae.js:290:37906)
	_layerAdd (urlParameters-2b23d0ae.js:16:69562)
	whenReady (urlParameters-2b23d0ae.js:16:44489)
	addLayer (urlParameters-2b23d0ae.js:16:69844)
	addTo (urlParameters-2b23d0ae.js:16:68896)
	onAdd (urlParameters-2b23d0ae.js:290:37589)
	_layerAdd (urlParameters-2b23d0ae.js:16:69562)
	whenReady (urlParameters-2b23d0ae.js:16:44489)
	addLayer (urlParameters-2b23d0ae.js:16:69844)
	(anonymous function) (index-fbed9313.js:754:33253)
	(anonymous function) (urlParameters-2b23d0ae.js:308:4631)
	(anonymous function) (urlParameters-2b23d0ae.js:290:37466) 
```

**Comment by mmd-osm** [comment]
Thank you, I could reproduce the issue for a few minutes only, but now it's working ok again.

Note that this issue was originally created for "Error: Invalid LatLng object: (NaN, NaN)". which causes a stuck UI, but likely for different reasons.

**Comment by gy-mate** [comment]
> Note that this issue was originally created for "Error: Invalid LatLng object: (NaN, NaN)". which causes a stuck UI, but likely for different reasons.

@mmd-osm Oh, thanks for the heads-up! I've opened a new issue for this one: #686
